python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py

```diff
@@ -1,278 +1,463 @@
+
+ot.Log.Show(ot.Log.NONE)
 
 # %%
-# Set the calibration parameters
-# ------------------------------
+# Define the observations
+# -----------------------
+# In practice, we generally use a data set which has been obtained from
+# measurements.
+# This data set can be loaded using e.g. :meth:`~openturns.Sample.ImportFromCSVFile`.
+# Here we define the data directly.
+cm = chaboche_model.ChabocheModel()
+print(cm.data)
+observedStrain = cm.data[:, 0]
+observedStress = cm.data[:, 1]
+nbobs = cm.data.getSize()
 
 # %%
-# Define the value of the reference values of the :math:`\theta` parameter. In the bayesian framework, this is called the mean of the *prior* gaussian distribution. In the data assimilation framework, this is called the *background*.
+# Print the Chaboche model
+print("Inputs:", cm.model.getInputDescription())
+print("Outputs:", cm.model.getOutputDescription())
 
 # %%
+# We see that there are four inputs: `Strain`, `R`, `C` and `Gamma` and one
+# output: `Sigma`.
+# The `Strain` is observed on input and the stress `Sigma` is observed on
+# output.
+# Using these observations, we want to estimate the parameters `R`, `C`
+# and `Gamma`.
+# %%
+# Set the calibration parameters
+# ------------------------------
+# In this part, we begin the calibration study.
+# Define the value of the reference values of the :math:`\theta` parameter.
+# In the Bayesian framework, this is called the mean of the *prior* Gaussian
+# distribution. In the data assimilation framework, this is called the *background*.
 R = 700e6  # Exact : 750e6
 C = 2500e6  # Exact : 2750e6
 Gamma = 8.0  # Exact : 10
 thetaPrior = [R, C, Gamma]
 
+# %%
+# In the physical model, the inputs and parameters are ordered as
+# presented in the next table.
+# Notice that there are no parameters in the physical model.
+#
+# +-------+----------------+
+# | Index | Input variable |
+# +=======+================+
+# | 0     | Strain         |
+# +-------+----------------+
+# | 1     | R              |
+# +-------+----------------+
+# | 2     | C              |
+# +-------+----------------+
+# | 3     | Gamma          |
+# +-------+----------------+
+#
+# +-------+-----------+
+# | Index | Parameter |
+# +=======+===========+
+# | ∅     | ∅         |
+# +-------+-----------+
+#
+# **Table 1.** Indices and names of the inputs and parameters of the physical model.
+#
+print("Physical Model Inputs:", cm.model.getInputDescription())
+print("Physical Model Parameters:", cm.model.getParameterDescription())
 
 # %%
-# The following statement create the calibrated function from the model. The calibrated parameters `R`, `C`, `Gamma` are at indices 1, 2, 3 in the inputs arguments of the model.
+# In order to perform calibration, we have to define a parametric model,
+# with observed inputs and parameters to calibrate.
+# In order to do this, we create a :class:`~openturns.ParametricFunction` where the parameters
+# are `R`, `C` and `Gamma` which have the indices 1, 2 and 3 in the physical model.
```

Comment:
It would be better to use :math: here and at other places for the sake of consistency. And LaTeX notations can also be used in plot legends.

Reply:
OK, I understand and agree with your logic. Maybe (but this PR is not the place), in the usecases module, "Gamma" should be replaced with "gamma" to fit the lowercase greek letter.

Reply:
Your suggestion on the Gamma greek letter makes sense, because the mathematical description shows a lower greek letter, but the Python code uses the "Gamma" string, which refer tot the upcase greek letter. Notice, however, that would create an inconsistency in the code where all the variables have a name where the first letter is upcase:
```python
  self.model.setInputDescription(["Strain", "R", "C", "Gamma"])
  self.model.setOutputDescription(["Sigma"])
  self.data.setDescription(["Strain", "Stress (Pa)"])
```
So, at the price of a small inconsistency, I would rather leave the code as is. Do you agree on that choice?

Reply:
OK, agreed.